tools/bazar/services/EntryManager.php

```diff
@@ -639,7 +639,8 @@ public function appendDisplayData(&$fiche, $semantic = false, $correspondance =
 
         // Données sémantiques
         if ($semantic) {
-            $form = $this->getService(FormManager::class)->getOne($fiche['id_typeannonce']);
+            // not possible to init the formManager in the constructor because of circular reference problem
+            $form = $GLOBALS['wiki']->services->get(FormManager::class)->getOne($fiche['id_typeannonce']);
```

Comment:
```suggestion
            $form = $this->wiki->services->get(FormManager::class)->getOne($fiche['id_typeannonce']);
```
est-ce plus élégant ?

Reply:
carrément ! j'avais vu que tu l'avais fait et je voulais le faire, j'ai juste zappé au dernier moment. Je refais un commit qui modifie aux 3 endroits.